pgjdbc/src/main/java/org/postgresql/core/v3/SimpleParameterList.java

```diff
@@ -312,6 +312,12 @@ public String toString(@Positive int index, boolean standardConformingStrings) {
           type = "box";
           break;
 
+        case Oid.BYTEA:
+          StreamWrapper wrapper = (StreamWrapper) paramValue;
+          textValue = ByteConverter.bytesToString(wrapper.getBytes());
+          type = "bytea";
+          break;
+
```

Comment:
虽然目前我们在此仓库还没有类似 MATRIXDB_VERSION 这样的宏或者代码标记机制, 不过建议至少以注释的形式把每一块咱们增强或修改的代码都加上 MatrixDB 这样的标记吧, 比如:

```suggestion
        /* MatrixDB: Allow expansion of bytea data */
        case Oid.BYTEA:
          StreamWrapper wrapper = (StreamWrapper) paramValue;
          textValue = ByteConverter.bytesToString(wrapper.getBytes());
          type = "bytea";
          break;
```